apps/api/src/token_deployment.py

```diff
@@ -33,39 +40,56 @@ async def deploy_token(name, ticker) -> tuple[str, list[dict]]:
     deployer_address = account.address
 
     # Deploy contract
-    contract = w3.eth.contract(abi=contract_abi, bytecode=contract_bytecode)
+    bonding_contract = w3.eth.contract(  # type: ignore
```

Comment:
Could you explain why the contract deployment looks so different now? I don't really understand what's going on here